salt/states/x509.py

```diff
@@ -522,6 +526,10 @@ def certificate_managed(name, days_remaining=90, append_certs=None, **kwargs):
         A list of certificates to be appended to the managed file.
         They must be valid PEM files, otherwise an error will be thrown.
 
+    managed_private_key:
+        Has no effect and will be removed in Salt Magnesium.
```

Comment:
Is it worth putting a "since v2016" to help avoid confusion about when it happened?

Reply:
Nice idea, updated now 👍